frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -53,15 +51,18 @@ function onClick(store: Store) {
 
     let serviceAreas = store.get('serviceAreas')
     let data = serviceAreas.map(_ => {
-      let representativePoint = representativePointsFromServiceAreas([_], store).value()[0]
+      let representativePoints = representativePointsFromServiceAreas([_], store).value()
+      if (representativePoints.length > 15000) {
+        alert('There are many population points in this state! Preparing the file may take a minute or two...')
```

Comment:
This is fine for now but we should really start using lightboxes for these kinds of messages.

Reply:
Is that easy to do?